source/_integrations/alert.markdown

```diff
@@ -269,6 +277,10 @@ alert:
     data:
       inline_keyboard:
         - 'Close garage:/close_garage, Acknowledge:/garage_acknowledge'
+      subtitle: >-
+        {% if states('sensor.garage_doors_open_count') | int(0) > 0 %}
+        {{ states('sensor.garage_doors_open_count') }} garage doors have been left open!
+        {% endif %}
     notifiers:
```

Comment:
To solve the lint errors put the yaml code block between `{% raw %}` and `{% enndraw %}`.

So:

```
{% raw %}

Your YAML code block

{% endraw %}
```

Reply:
Thanks! I'll push another revision up soon, probably when I hear a bit about the main pull request.